common/exit-codes.h

```diff
@@ -0,0 +1,11 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+enum class ExitCode {
```

Comment:
Please comment magic constants and logic behind it.

Reply:
Fixed